### PR TITLE
Aks default vm size

### PR DESCRIPTION
**Pull request by seanknox**
This changes the default VM size to Standard_DS2_v2 when running `az aks create`.

@sauryadas @seanmck for viz.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by seanknox** [comment]
@troydai could I get a review?

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -5,8 +5,10 @@ Release History
 
 2.3.1
 +++++
+* `az aks create` now defaults to Standard_DS2_v2 VMs.
```
Comment (troydai) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: You need to bump the version as 2.3.1 just released.

Comment (seanknox) [reply src/command_modules/azure-cli-acs/HISTORY.rst]: Done, thanks!

Comment (troydai) [reply src/command_modules/azure-cli-acs/HISTORY.rst]: As well as in the `setup.py`.

Comment (seanknox) [reply src/command_modules/azure-cli-acs/HISTORY.rst]: Done.

**Pull request merged**